http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/DefaultServiceAccountProvider.scala

```diff
@@ -1,28 +0,0 @@
-package org.broadinstitute.dsde.workbench.leonardo.auth
```

Comment:
this isn't used anywhere I'm fairly sure, and it makes figuring out whats going on with service accounts hard